code/modules/vehicles/lavaboat.dm

```diff
@@ -63,11 +64,12 @@
 	name = "mysterious boat"
 	desc = "This boat moves where you will it, without the need for an oar."
 	icon_state = "dragon_boat"
+	fb = 85
 
 /obj/vehicle/ridden/lavaboat/dragon/Initialize()
 	. = ..()
 	var/datum/component/riding/D = LoadComponent(/datum/component/riding)
 	D.vehicle_move_delay = 1
 	D.set_riding_offsets(RIDING_OFFSET_ALL, list(TEXT_NORTH = list(1, 2), TEXT_SOUTH = list(1, 2), TEXT_EAST = list(1, 2), TEXT_WEST = list( 1, 2)))
 	D.keytype = null
-	AddComponent(/datum/component/fishingbonus,80)
+	AddComponent(/datum/component/fishingbonus, fb)
```

Comment:
Dont need this line since it's handled in /obj/vehicle/ridden/lavaboat/Initialize() and that's called at the ..()

Reply:
ok i thought it wouldnt do it